target-app/Home/Views/HomeViewController.swift

```diff
@@ -113,6 +115,16 @@ class HomeViewController: UIViewController {
             .sink { [weak self] location in
                 self?.createAnotation(withLocation: location)
             }.store(in: &cancellables)
+        
+        viewModel.$targets
+            .sink{ [weak self] targets in
+                if let targets = targets {
+                    targets.forEach { target in
+                        let location = CLLocation(latitude: target.target.latitude, longitude: target.target.longitude)
```

Comment:
this is what I was trying to tell in my previous comment. If we use just `Target` objects this could be less reduntant.